docs/Changelog.md

```diff
@@ -18648,6 +18648,11 @@ This version of the operator has been available since version 14 of the default
   
   current_mean = ReduceMean(X, axis=all_except_channel_index)
   current_var =  ReduceVar(X, axis=all_except_channel_index)
+  
+  Notice that ReduceVar refers to the population variance, and it equals to
+  sum(sqrd(x_i - x_avg)) / N
+  where N is the population size (this formula does not use sample size N - 1).
```

Comment:
Minor nit: I would suggest "where the denominator is N and not (N-1), where N is the number of summed elements." If we want to contrast with sample variance, suggest changing "refers to the population variance" to "refers to the population variance, and not sample variance".

Reply:
Sorry about the comment on the wrong PR! I should have added this to the original PR.